Utils/ContextToYML.yml

```
@@ -0,0 +1,202 @@
+  outputs:
+  - contextPath: Demisto.Id
+    description: ''
+    type: Number
+  - contextPath: Demisto.Description
+    description: ''
+    type: String
+  - contextPath: Demisto.Summary
+    description: ''
+    type: String
+  - contextPath: Demisto.Created
+    description: ''
+    type: String
+  - contextPath: Demisto.Issuetype.Id
+    description: ''
+    type: Number
+  - contextPath: Demisto.Issuetype.Name
+    description: ''
+    type: String
+  - contextPath: Demisto.Project.Id
+    description: ''
+    type: Number
+  - contextPath: Demisto.Project.Key
+    description: ''
+    type: String
+  - contextPath: Demisto.Project.ProjectTypeKey
+    description: ''
+    type: String
+  - contextPath: Demisto.Status.Id
+    description: ''
+    type: Number
+  - contextPath: Demisto.Status.StatusCategory.Key
+    description: ''
+    type: String
+  - contextPath: Demisto.Status.StatusCategory.Colorname
+    description: ''
+    type: String
+  - contextPath: Demisto.Status.StatusCategory.Name
+    description: ''
+    type: String
+  """
+
+  VERBOSE = demisto.getArg('verbose') == 'true'
+  CAMELIZE = demisto.args().get('camelize', 'false') == 'true'
+
+
+  def flatten_json(nested_json):
+      out = {}
+
+      def flatten(x, name=''):
+          # capitalize first letter in each key
+          try:
+              name = name[0].upper() + name[1:] if CAMELIZE else name
+          except IndexError:
+              name = name.title() if CAMELIZE else name
+
+          if isinstance(x, dict):
+              for a in x:
+                  flatten(x[a], name + a + '.')
+          elif isinstance(x, list):
+              for a in x:
+                  flatten(a, name[:-1] + '.')
+          else:
+              out[name.rstrip('.')] = x
+
+      flatten(nested_json)
+      return out
+
+
+  def jsonise(context_key, value):
+      return {'contextPath': context_key, 'description': '', 'type': determine_type(value)}
+
+
+  def determine_type(val):
+      return 'Boolean' if isinstance(val, bool) else 'Number' if isinstance(
+          val, (int, float)) else 'String' if isinstance(val, str) else 'Unknown'
+
+
+  def parse_json(data, command_name, base_path):
+      flattened_data = flatten_json(data)
+      if base_path:
+          flattened_data = {f'{base_path.title()}.{key}': value for key, value in flattened_data.items()}
+
+      arg_json = [jsonise(key, value) for key, value in flattened_data.items()]
+
+      if VERBOSE:
+          print(f'JSON before converting to YAML: {arg_json}')
+
+      yaml_output = yaml.safe_dump(
+          {
+              'name': command_name.lstrip('!'),
+              'arguments': [],
+              'outputs': arg_json
+          },
+          default_flow_style=False
+      )
+      return yaml_output
+
+
+  def main():
+      command_name = demisto.args().get("command_name", '')
+      base_path = demisto.args().get("base_path", '')
+      entry_id = demisto.getArg("json_file_entry_id")
```

save `demisto.args()` as a variable instead of calling it each time.
```suggestion
      entry_id = demisto.args().get('json_file_entry_id')
```